.github/workflows/main.yml

```diff
@@ -0,0 +1,101 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Build pdf
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the main branch
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+  - cron: "0 12 1 * *"
```

Comment:
Hello!

Why is this needed?

Reply:
Hmmm, in fact. It might be useless. 
I did not GitHub actions when I have tried all of this.

We should remove this. I'm doing it